vendor/github.com/prometheus/tsdb/querier.go

```diff
@@ -53,6 +53,9 @@ type Series interface {
 
 	// Iterator returns a new iterator of the data of the series.
 	Iterator() SeriesIterator
+
+	// Chunks returns a copy of the compressed chunks that make up this series.
+	Chunks() []chunks.Meta
```

Comment:
Where is this coming from? I cannot see this on tip of TSDB :thinking: nor 0.8.0 which `go mod` vendors here. 

Is this manually modifed TSDB code in vendor? Cannot see any `replace` directive on `go.mod` as well so you don't use fork as well...

Reply:
I just added it here; opening a TSDB PR now.

Reply:
https://github.com/prometheus/tsdb/pull/659